source/includes/usage-examples/write-code-examples.c

```diff
@@ -0,0 +1,142 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int
+main (int argc, char *argv[])
```

Comment:
S: same suggestion here about void
```suggestion
main (void)
```